packages/twenty-front/src/modules/object-record/record-table/components/RecordTable.tsx

```diff
@@ -44,6 +46,11 @@ export const RecordTable = () => {
   const recordTableIsEmpty =
     !isRecordTableInitialLoading && allRecordIds.length === 0;
 
+  const isFocusActive = useRecoilComponentValueV2(
```

Comment:
Let's follow the naming pattern here, state name equals variable name without the state suffix